sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -57,9 +61,17 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
 
       tags ++ branches
     },
+    tlCreateJobSummary := {
```

Comment:
Unfortunately this name is a bit too vague—in sbt, all keys exist in global scope, so the name must be globally unique across all sbt plugins. That's why all the keys are prefixed with `tl` (typelevel), and then I also try to include the name of the specific sub-plugin. e.g. `tlCiReleaseStepSummary`, since this is the CI (sonatype) release plugin.

Reply:
Makes sense